Mono.Debugging/Mono.Debugging.Evaluation/NRefactoryExpressionEvaluatorVisitor.cs

```diff
@@ -68,6 +68,10 @@ static string ResolveTypeName (SyntaxNode type)
 			string name = type.ToString ();
 			if (name.StartsWith ("global::", StringComparison.Ordinal))
 				name = name.Substring ("global::".Length);
+
+			var indexOfSlash = name.IndexOf('<');
```

Comment:
slash? :D

Reply:
heh... I remember when I wrote this writing a `/` and replacing it with `<` :)

Reply:
renamed to index :)